packages/model-viewer/src/three-components/PlacementBox.ts

```diff
@@ -105,6 +106,12 @@ export class PlacementBox extends Mesh {
     (this.hitPlane.material as Material).side = DoubleSide;
     this.add(this.hitPlane);
 
+    this.hitBox = new Mesh(new BoxGeometry(
+        size.x + 2 * RADIUS, size.y + RADIUS, size.z + 2 * RADIUS));
```

Comment:
I didn't understand these values for the box. Maybe a comment or function would help

Reply:
Fair, that's a bit confusing.